docs/src/modules/components/AppTableOfContents.js

```diff
@@ -114,9 +115,21 @@ function flatten(headings) {
   return itemsWithNode;
 }
 
+const shouldShowJobAd = () => {
+  const date = new Date();
+  const timeZoneOffset = date.getTimezoneOffset();
+  // Hide for IST (GMT+5:30)
```

Comment:
Should we hide for SF and Japan/Australia kind of timezones?

Reply:
Yeah, let's do that. I will update to hide for: 
> UTC+5.5  (India, Japan, Australia etc.)
and < UTC-8 (California, Alaska, New Zealand etc.)